lib/ReaderCollectionPrioritized.js

```diff
@@ -33,6 +39,9 @@ class ReaderCollectionPrioritized extends AbstractReader {
 	 * @returns {Promise<@ui5/fs/Resource[]>} Promise resolving to list of resources
 	 */
 	_byGlob(pattern, options, trace) {
+		if (this._emptyCollection) {
```

Comment:
Not sure whether there is really much overhead when `_readers` is empty.
Also, currently `trace.collection` won't be called anymore. Not sure whether this causes issues with tracing.

Reply:
Done